less/mixins.less

```diff
@@ -1 +1,24 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.custom-btn() {
+    display: block;
+    cursor: pointer;
+    height:40px;
```

Comment:
You could also add some parameters to make your button mixin even more versatile.